.github/workflows/attestation.yml

```diff
@@ -10,6 +10,16 @@ on:
         type: string
         description: "Architecture"
         required: true
+      version:
+        type: string
+        description: 'crane release to be installed'
+        required: false
+        default: v0.20.5
```

Comment:
Maybe we could use env at job level in the "Install the crane command" step instead of modifying the action inputs.

e.g.

```
jobs:
  attest:
    runs-on: ubuntu-latest
    env:
      CRANE_VERSION: v0.20.5
      CRANE_CHECKSUM: ad4cd9af2568c62c97e346de6d1295ee8c6ce3341f7b71cf02d41292b4532680
      [...]
```